libkbfs/data_types.go

```diff
@@ -565,6 +565,14 @@ type EntryInfo struct {
 	Ctime int64
 }
 
+// EntryInfoExtended extends the EntryInfo with some locally computed
+// attributes.
+type EntryInfoExtended struct {
```

Comment:
I was thinking the writable permission is platform-indepedent, and putting it in libkbfs means we can reuse the same code on other platforms as well. But I'm OK with moving these to libfuse. Will update; thanks!